motoko/cert-var/src/cert_var_assets/src/index.js

```diff
@@ -25,15 +25,67 @@ document.getElementById("setBtn").addEventListener("click", async () => {
 
   log.innerText = "Verifying gotten value " + newVal + "from " + cid + "...";
 
+  /*
+    # Response certification example: Single 32-bit Variable.
+
+    To detect an attacker in the middle between us and the IC (and our
+    "true" canister running there), we must either:
+
+    - perform update calls that use "full consensus" (and wait for ~2 sec).
+    - perform (fast) query calls whose responses that we, the client, certify,
+      using the coordination of the IC and our canister running there.
+
+    This code demonstrates the second approach here, in a minimal setting.
+
+    The full example (beyond this file) consists of a Motoko canister
+    (called cert_var here) that holds a single certified variable, as
+    a 32-bit number, and an asset canister whose (client-side) code
+    (here!) queries and certifies this number's "current certificate".
+
+    The Motoko backend canister prepares for our certification here by
+    giving us a "current certificate" within the response; this certificate
+    is signed by the entire IC, using a system feature described here:
+
+    https://sdk.dfinity.org/docs/interface-spec/index.html#system-api-certified-data
+
+    Before we trust the response from our apparently "true" canister,
+    we interrogate it, and verify its authenticity:
+
+    We do four checks below:
+
+    1. verify system certificate.
+    2. check system certificate timestamp is not "too old".
+    3. check canister ID in system certificate.
+    4. check response matches witness.
+
+    For steps 2, 3 and 4, we access data from the certificate (Blob).
+    The Certificate class from the agent-js library provides a way to
+    access those items using their paths, like a filesystem, each addressing
+    a Blob, encoding something.
+
+    In the case of time and our data, the encodings are each Candid.
```

Comment:
The spec says `time` is using the LEB128 encoding, and our certified data is represented in little endian. Ideally, we should use a proper library to decode these numbers. To prevent an extra dependency, we take advantage of the fact that the Candid value encoding of `nat` and `nat32` happens to use the same representation we need.